tracing.go

```diff
@@ -22,32 +22,37 @@ package tchannel
 
 import (
 	"fmt"
-	"math/rand"
+	"net"
+	"strconv"
+	"time"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"golang.org/x/net/context"
```

Comment:
is it codified somewhere in this repo?  The order I use is (1) std libs, (2) other external deps, (3) this repo own packages